frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -154,6 +155,7 @@ public BranchOrCommitCellRendererComponent(
 
       if (commitItem.getCommit().equals(forkPoint)) {
         append(" ${HEAVY_WIDE_HEADED_RIGHTWARDS_ARROW} fork point ? ", new SimpleTextAttributes(STYLE_PLAIN, Colors.RED));
+        append("(Resolve) ", LINK_ATTRIBUTES);
```

Comment:
Nit: the underlined trailing whitespace doesn't look good in links in general

Reply:
Also, the link's not working as for now ;D

Reply:
whooops xd